vendor/engines/c2po/spec/controllers/account_price_group_members_controller_spec.rb

```diff
@@ -8,14 +8,36 @@
   describe "search_results" do
     # Ignore validation errors, e.g. number format
     before { allow(AccountValidator::ValidatorFactory).to receive(:instance).and_return(AccountValidator::ValidatorDefault.new) }
+    before { skip if facility_account_type.blank? }
 
     let(:partial_account_number) { build(:nufs_account).account_number[0..-4] }
     let(:price_group) { create(:price_group, facility: facility) }
-    let!(:global_account) { create(:nufs_account, :with_account_owner, account_number: "#{partial_account_number}234") }
-    let!(:facility_purchase_order) { create(:purchase_order_account, :with_account_owner, account_number: "#{partial_account_number}894", facility: facility) }
-    let!(:other_facility_purchase_order) { create(:purchase_order_account, :with_account_owner, account_number: "#{partial_account_number}542", facility: create(:facility)) }
-
-    let(:user) { create(:user, :facility_administrator, facility: facility) }
+    let(:global_account_type) { Account.config.global_account_types.first }
+    let(:facility_account_type) { Account.config.facility_account_types.first }
+    let!(:global_account) do
+      create(
+        global_account_type.underscore,
```

Comment:
Should we check that the factory exists?

Reply:
i can add a workaround for that case